source/_integrations/alarm_control_panel.mqtt.markdown

```diff
@@ -205,3 +205,20 @@ value_template:
   required: false
   type: template
 {% endconfiguration %}
+
+## Example
+
+```yaml
+# Example configuration.yaml entry
+alarm_control_panel:
+  - platform: mqtt
+    name: My Alarm
```

Comment:
Make sure this example is formatted according to our YAML style guide here:

<https://developers.home-assistant.io/docs/documenting/yaml-style-guide>